refinery/tool_manager/utils.py

```diff
@@ -300,24 +352,24 @@ def validate_tool_launch_configuration(tool_launch_config):
     with open("tool_manager/schemas/ToolLaunchConfig.json") as f:
         schema = json.loads(f.read())
     try:
-        validate(
-            json.loads(tool_launch_config),
-            schema,
-            resolver=resolver
-        )
-    except ValueError as e:
-        raise RuntimeError(
-            "Tool launch configuration: {} is not valid JSON: {}".format(
-                tool_launch_config,
-                e
-            )
-        )
+        validate(tool_launch_config, schema, resolver=resolver)
     except ValidationError as e:
```

Comment:
Again, I think catching a specific error, then raising Runtime error with a useful message is helpful here